cmd/run.go

```diff
@@ -307,22 +307,13 @@ func (c *cmdRun) run(cmd *cobra.Command, args []string) (err error) {
 	// Init has passed successfully, so unless disabled, make sure we send a
 	// usage report after the context is done.
 	if !conf.NoUsageReport.Bool {
-		backgroundProcesses.Add(2)
-		reportCtx, reportCancel := context.WithCancel(globalCtx)
-		reportDone := make(chan error)
+		backgroundProcesses.Add(1)
+		reportCtx, reportCancel := context.WithTimeout(globalCtx, 3*time.Second)
```

Comment:
Can you move this line to below the first `defer` 

We do not need the new context outside of that funciton and I kind of prefer code where if there is somethign that it will be `defer`ed it is *right* after it needs to be.

Reply:
Resolved by [this force-push](https://github.com/grafana/k6/compare/1776af241bc048772c2042bea4e1a3e284f66114..635d0815c9b117aa79f213fccb12876fff79f134).